src/main/java/com/eppo/sdk/helpers/bandit/FalconBanditModel.java

```diff
@@ -0,0 +1,92 @@
+package com.eppo.sdk.helpers.bandit;
+
+import com.eppo.sdk.dto.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FalconBanditModel implements BanditModel {
+
+  public static final String MODEL_IDENTIFIER = "falcon";
+
+  public Map<String, Double> weighActions(BanditParameters parameters, Map<String, EppoAttributes> actions, EppoAttributes subjectAttributes) {
+
+    // For each action we need to compute its score using the model coefficients
+    Map<String, Double> actionScores = actions.entrySet().stream().collect(Collectors.toMap(
+      Map.Entry::getKey, e -> {
+        String actionName = e.getKey();
+        EppoAttributes actionAttributes = e.getValue();
+        double actionScore = 0.0;
```

Comment:
(Outside of this PR, maybe.) We need to have this default score as a parameter instead of a hardcoded 0.

The issue is unknown actions. Depending on the range of values for the optimization metric (and hence model scores for _known_ actions), an unknown action with a default value of `0.0` might end up essentially never being chosen (if the model scores are very large), or in rare cases chosen too often (if the model scores are somehow negative).

When all actions are present from the beginning of the bandit's life, there is no problem. But when new actions appear over time, it could be an issue.  

To solve this, we'll want to set a `defaultActionScore` as part of the post-training process, and set it at some reasonable value (e.g. at the 10th percentile of the distribution of outcomes). I don't know yet exactly how we'll set that value (in fact, we may just start with 0), but we should start planning for it in the SDK. It can go alongside the `gamma` parameter.

Reply:
This all makes sense to me; I'll add a spot for it in the RAC and use it!

Reply:
Excellent, yes we need to think about this more -- we should be normalizing the outcomes such that the average outcome is 0 across all actions, but if there are some terrible actions and some really strong ones, then the unknown actions are not shown at all.

Adding a defaultActionScore param gives us the flexibility to figure out what to do here, though we still have to think carefully about what the best approach would be.

Reply:
That's right – I forgot about the demeaning step. Good point. The extra flexibility should be useful regardless.